qml/flightreservation.h

```diff
@@ -50,6 +50,9 @@ class FlightReservation : public QObject
         QString arrivalAirportCode() const;
         QDateTime arrivalTime() const;
 
+    public Q_SLOTS:
+        void setMap(QVariantMap& map);
+
```

Comment:
From the plasmoid, I will do something like : 
FlightReservation{
id : myOb
}
onFlightDataReceived : myOb.setMap(m_map)